LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017-2018 Nikolay Kim & contributors
```

Comment:
I think it's better to add `MIT License` in the first line.

Reply:
Got it